docs/admin-portal.md

```diff
@@ -21,6 +21,8 @@ You will require Node.js and `npm`. These instructions were tested with the vers
 
 1. Run `make build-portal` from the main directory. This will install the dependencies and kick off the Webpack build, placing the results in `portal/v2/build/`.
 
+  > __NOTE:__ Due to security compliance requirements, the `make build-portal` target pulls from `arointsvc.azurecr.io`. You can either authenticate to this registry using `az acr login --name arointsvc` to pull the image, or modify the Dockerfiles locally to point to `registry.access.redhat.com` instead.
```

Comment:
Can this be updated to reflect the changes to the Makefile?, same with the other doc `deploy-full-rp-service-in-dev.md`

Reply:
Done, let me know if this is what you were thinking.

Reply:
Yeah that is perfect, thank you